tools/amd_build/build_amd.py

```diff
@@ -141,10 +135,17 @@
                     f.flush()
                     os.fsync(f)
 
+# Check if the compiler is hip-clang.
+def is_hip_clang():
+    try:
+        return 'HIP_COMPILER=clang' in open('/opt/rocm/hip/lib/.hipInfo').read()
```

Comment:
We should avoid checking absolute path. It may conflicts with custom environment from a developer.

Reply:
even though that works for the current CI setting, it may conflict with other developers environment

Reply:
updated to use HIP_PATH environment variable.